lib/v2/blocs/authentication/viewmodels/authentication_bloc.dart

```diff
@@ -30,6 +30,12 @@ class AuthenticationBloc extends Bloc<AuthenticationEvent, AuthenticationState>
       // Set fcm token must be last instruction to allow login, even if there is an error here.
       await FirebaseMessageTokenRepository().setFirebaseMessageToken(event.account);
     }
+    if (event is OnCreateAccount) {
```

Comment:
this mirrors the OnImportEvent above it except user and private key have already been stored and the user still needs to back up